test/tests/meshgenerators/block_deletion_generator/tests

```diff
@@ -33,6 +33,13 @@
       recover = false
       detail = 'blocks and assigning the new boundary even if the input mesh is not prepared.'
     []
+    [delete_interior_parents]
+      type = 'RunApp'
```

Comment:
this should be an exodiff.
We need to make sure we get the right result out of this